README.md

```diff
@@ -1,56 +1,125 @@
 # cf-plugin-backup
 A Cloud Foundry Plugin that allows backup and restore of the CCDB using CF API.
 
+## Limitations
+
+   - The basic use case for the plugin is saving the state of a CF
```

Comment:
Even though it doesn't appear in the rendered README, it looks like the indentation is off slightly between paragraphs.

Otherwise LGTM.

Reply:
Checking ...

Reply:
Confirmed, 2nd to last item were only indented 2 spaces instead of 3.
Fixed with commit 8ae9bc013d08b9d425bf5dedb28c3b5b5052eb2d